cdhweb/pages/models.py

```diff
@@ -291,6 +292,15 @@ def get_context(self, request):
         # add up to 3 upcoming, published events
         context["events"] = Event.objects.live().upcoming()[:3]
 
+        # add "featured pages" with special section: currently about/consult,
+        # don't add them to context if not published
+        # NOTE effectively hardcoding by slug for now; could generalize later
+        context["about"] = ContentPage.objects.live().filter(slug="about").first()
+        context["consult"] = ContentPage.objects.live().filter(slug="consult").first()
+
+        # about page image to be used for featured page section
+        context["featured_img"] = Image.objects.filter(tags__name="homepage").first()
```

Comment:
this seems a little strange to me, and potentially unreliable

Reply:
I think the image should either be included as an asset in `sitemedia` or it should be configured via an image chooser field/panel on the page it is associated with. I only see you using one size in the template, so I think the simplest thing right now is just to put it in the sitemedia folder. When/if we decided it needs to be configurable we can switch it to a featured image field on the page — but that doesn't seem like it's worth spending time on right now.